expressions/aggregation/AggregationConcreteHandle.hpp

```diff
@@ -27,10 +27,12 @@
 #include "catalog/CatalogTypedefs.hpp"
 #include "expressions/aggregation/AggregationHandle.hpp"
 #include "storage/HashTable.hpp"
+#include "storage/FastHashTable.hpp"
```

Comment:
As a convention, we sort the included files in alphabetical order. If you use vim, there's an easy way to do that. You can select all the #include statements in visual mode and type sort.